features/us55-borrower-verified-landing-page.feature

```diff
@@ -0,0 +1,18 @@
+@us55
+
+Feature: Check content on Borrower Landing Page
+
+Acceptance Criteria
+    Page content matches wireframe
+
+Scenario Outline: Borrower arrives landing Page
+    Given I reach the Borrower Identity Verified Page
+    Then I should see <Link_button>
+    And the following <Texts>
```

Comment:
I would also be tempted to reword the And statement to be a bit more descriptive, something like 'And the following <help paragraphs> are displayed' or along those lines, just to make it a bit more readable to people outside of the team.